examples/multiple-endpoints/main.tf

```diff
@@ -0,0 +1,220 @@
+provider "aws" {
+  region = "eu-west-1"
+  alias  = "eu"
+}
+
+provider "aws" {
+  region = "us-west-1"
+  alias  = "us"
+}
+
+locals {
+  name = "ga-${replace(basename(path.cwd), "_", "-")}"
+
+  tags = {
+    Name       = local.name
+    Example    = local.name
+    Repository = "https://github.com/terraform-aws-modules/terraform-aws-global-accelerator"
+  }
+}
+
+################################################################################
+# Global Accelerator Module
+################################################################################
+
+module "global_accelerator" {
+  source = "../.."
+
+  name = local.name
+
+  # Listeners
+  listeners = {
+    listener_1 = {
+      client_affinity = "SOURCE_IP"
+
+      endpoint_groups = {
```

Comment:
Let's add support for the new `endpoint_groups` like you do in this PR, and have support for `endpoint_group` (previous implementation, see `examples/complete`).

Reply:
Hi @antonbabenko, the module had not _explicitly_ supported the variable `endpoint_group` nor `endpoint_groups`. The `endpoint_group(s)` _field_ had been passed _implicitly_ via `listeners` variable instead.

I think we should have **one** `endpoint_groups` option for the `listeners` variable. This is "less confusing and more readable" than having 2 different fields to deal with. 

For the Single Endpoint scenario, user only need to pass "one single endpoint" in the `endpoint_groups` field like this

```hcl
 endpoint_groups = {
        one = {
          endpoint_group_region         = "eu-west-1" # Optional 
          health_check_port             = 80
          health_check_protocol         = "HTTP"
          health_check_path             = "/"
          health_check_interval_seconds = 10
          health_check_timeout_seconds  = 5
          threshold_count               = 2
        }
```

I can also replace the "complete" example with the scenario "single-endpoint" so that user can know how to do with single endpoint. 

Let me know your thought